tests/test/org/commcare/cases/CaseTestUtils.java

```
@@ -0,0 +1,58 @@
+package org.commcare.cases;
+
+import org.commcare.test.utils.TestInstanceInitializer;
+import org.commcare.util.mocks.MockUserDataSandbox;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.ExternalDataInstance;
+import org.javarosa.model.xform.DataModelSerializer;
+import org.javarosa.xpath.XPathParseTool;
+import org.javarosa.xpath.expr.XPathExpression;
+import org.javarosa.xpath.expr.XPathFuncExpr;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+public class CaseTestUtils {
+    public static final String CASE_INSTANCE = "jr://instance/casedb";
+    public static final String LEDGER_INSTANCE = "jr://instance/ledgerdb";
+
+    public static byte[] loadCaseInstanceIntoSandbox(MockUserDataSandbox sandbox) {
+        try {
+            ByteArrayOutputStream bos = new ByteArrayOutputStream();
+            DataModelSerializer s = new DataModelSerializer(bos, new TestInstanceInitializer(sandbox));
+
+            s.serialize(new ExternalDataInstance(CASE_INSTANCE, "instance"), null);
+            return bos.toByteArray();
+        } catch (IOException e) {
+            throw new RuntimeException(e.getMessage());
+        }
+    }
+
+    public static boolean xpathEval(EvaluationContext evalContext,
+                                    String input,
+                                    Object expectedOutput) {
+        XPathExpression expr;
+        try {
+            expr = XPathParseTool.parseXPath(input);
+        } catch (XPathSyntaxException e) {
+            e.printStackTrace();
```

Hm, should this signature just throw these rather than swallow them? Otherwise the traces won't have them